cmd/openshift-installer/main.go

```diff
@@ -0,0 +1,49 @@
+package main
```

Comment:
nit: maybe just rename this folder to `cmd/openshift-install/main.go`, So the binary will be openshift-install.
Then we can say things like
`opensihft-install installconfig`
`opensihft-install manifests`
`opensihft-install ignconfigs`
`opensihft-install cluster`